apps/server/src/modules/board/controller/api-test/board-drawing-child.api.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
+import { ServerTestModule } from '@modules/server/server.module';
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { BoardExternalReferenceType } from '@shared/domain/domainobject';
+import {
+	cardNodeFactory,
+	cleanupCollections,
+	columnBoardNodeFactory,
+	columnNodeFactory,
+	courseFactory,
+	TestApiClient,
+	UserAndAccountTestFactory,
+} from '@shared/testing';
+import { drawingElementNodeFactory } from '@shared/testing/factory/boardnode/drawing-element-node.factory';
+
+const baseRouteName = '/boards';
+
+describe(`has drawing child (api)`, () => {
+	let app: INestApplication;
+	let em: EntityManager;
+	let testApiClient: TestApiClient;
+
+	beforeAll(async () => {
+		const module: TestingModule = await Test.createTestingModule({
+			imports: [ServerTestModule],
+		}).compile();
+
+		app = module.createNestApplication();
+		await app.init();
+		em = module.get(EntityManager);
+		testApiClient = new TestApiClient(app, baseRouteName);
+	});
+
+	afterAll(async () => {
+		await app.close();
+	});
+
+	const setup = async () => {
```

Comment:
the setup should normally not be for the whole file, but at describe level, so that we can easier see the diferences betweeen the various scenarious you describe